man/org.freedesktop.locale1.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" >
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="org.freedesktop.locale1" conditional='ENABLE_LOCALED'
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>org.freedesktop.locale1</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>org.freedesktop.locale1</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>org.freedesktop.locale1</refname>
+    <refpurpose>The D-Bus interface of systemd-localed</refpurpose>
+  </refnamediv>
+
+  <refsect1>
+    <title>Introduction</title>
+
+    <para>
+    <citerefentry><refentrytitle>systemd-localed.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    is a system service that can be used to control the system locale and keyboard mapping from user
+    programs. This page describes the D-Bus interface.</para>
+  </refsect1>
+
+  <refsect1>
+    <title>The D-Bus API</title>
+
+    <para>The service exposes the following interfaces on the bus:</para>
+
+    <programlisting>
+$ gdbus introspect --system \
+        --dest org.freedesktop.locale1 \
+        --object-path /org/freedesktop/locale1
+
+node /org/freedesktop/locale1 {
+  interface org.freedesktop.locale1 {
+    methods:
+      SetLocale(in  as locale,
+                in  b interactive);
+      SetVConsoleKeyboard(in  s keymap,
+                          in  s keymap_toggle,
+                          in  b convert,
+                          in  b interactive);
+      SetX11Keyboard(in  s layout,
+                     in  s model,
+                     in  s variant,
+                     in  s options,
+                     in  b convert,
+                     in  b interactive);
+    properties:
+      readonly as Locale = ['...', ...];
+      readonly s X11Layout = '...';
+      readonly s X11Model = '...';
+      readonly s X11Variant = '...';
+      readonly s X11Options = '...';
+      readonly s VConsoleKeymap = '...';
+      readonly s VConsoleKeymapToggle = '...';
+  };
+  interface org.freedesktop.DBus.Peer { ... };
+  interface org.freedesktop.DBus.Introspectable { ... };
+  interface org.freedesktop.DBus.Properties { ... };
+};
+    </programlisting>
+
+    <refsect2>
+      <title>Methods</title>
+
+      <para>If you set a new system locale all old system locale settings will be dropped, and the new
+      settings will be saved to disk. It will also be passed to the system manager, and subsequently started
```

Comment:
"settings" is better I think, because it's not just the locale, but additional things that all get saved in one atomic write...

Reply:
Alright but then I would use "the locale" instead of "It" in the next sentence because it was a bit unclear what "It" referred to because you used "settings" in the previous sentence
```suggestion
      settings will be saved to disk. The locale will also be passed to the system manager, and subsequently started
```